docs/en/pid-eaa-data-model.rst

```diff
@@ -7,8 +7,7 @@ PID/(Q)EAA Data Model
 +++++++++++++++++++++
 
 The Person Identification Data (PID) is issued by the PID Provider following national laws and allows a natural person to be authenitcated and identified. 
-
-The User attributes carried in the Italian PID are:
+The User attributes carried within the Italian PID are the ones listed below:
```

Comment:
In the new version of the ARF 1.2, the mandatory set of user attributes for the PID has been changed, do we need to be compliant with that? they considered `age_over_18` as a mandatory attribute in the Italian PID?

Reply:
good catch @asharif1990 

moved to a separate issue